### ISSUE TITLE
NUT 2.8 / 3.0: general tasks list

### ISSUE THREAD

**Issue by aquette** [networkupstools/nut#370]
This is an attempt to list the things to make NUT more generic for NUT 2.8 / 3.0:

* [ ] variable generalization: generalize some variables, which are valid for any device type and not only for ups.* (i.e. to move to device.*) (ups.firmware, ups.temperature, <to be completed>)
* [ ] "ups" pattern replacement: replace "ups" patterns by "nut" (upsc, upsrw, upscmd, upsd, upsmon, upssched => nutc, nutrw, nutcmd, nutd, nutmon, nutsched)
* [ ] to be continued...

These may be spawned into other issues, to ease work...

**Comment by clepple** [comment]
I don't believe that the potential benefit of renaming the utilities to eliminate "UPS" will outweigh the confusion during the transition.

For the variables, maybe we could add an alias feature to the dstate layer to publish in both locations for a while (the current method of duplicating variables at startup is hackish). I don't think that's a big win either, but it is not quite as drastic.
